include/envoy/grpc/status.h

```diff
@@ -5,9 +5,11 @@ namespace Grpc {
 
 class Status {
 public:
+  using GrpcStatus = int64_t;
+
   // If this enum is changed, then the std::unordered_map in Envoy::Grpc::Utility::nameToGrpcStatus
   // located at: //source/common/access_log/grpc/status.cc must also be changed.
-  enum GrpcStatus {
+  enum WellKnownGrpcStatus {
```

Comment:
Renaming this enum type will cause build problems for all extensions that use it (and live outside of the envoy repository). Is it necessary that the uint64_t alias be named GrpcStatus? Could it be Type? Grpc::Status::Type may look better than the Grpc::Status::GrpcStatus

Reply:
@yanavlasov we don't have C++ API stability. Renaming to Type also break the build outside of the tree too?

Reply:
I'm going to propose to have at least some common sense stability. We had to spend significant time fixing builds of internal filters for upstream C++ API changed that I think were totally avoidable.
Here I meant using Type in the int64_t alias and leaving enum GrpcStatus unchanged. I.e:
using Type = int64_t;
enum GrpcStatus { ... };

Reply:
yeah if it is avoidable. If we use `Type`, don't you still have to change all `Grpc::Status::GrpcStatus` to `Grpc::Status::Type`?

Reply:
I see. Yes, you are correct, with this change filters have to change anyway. I looked at the internal filters and I think the original change with using GrpcStatus for int64_t alias is less problematic. So, please disregard my comment.